tools/docker/docker-installation.md

```diff
@@ -53,4 +80,4 @@ $ sudo groupadd docker && sudo gpasswd -a ${USER} docker && sudo systemctl resta
 $ newgrp docker
 ```
 
-You have to log out and log back in (or restart Docker daemon and use `newgrp` command as mentioned here) for these changes to take effect. Then you can verify if your changes were successful by running Docker without `sudo`.
+You have to log out and log back in (or restart Docker daemon and use `newgrp` command as mentioned here) for these changes to take effect. Then you can verify if your changes were successful by running Docker without `sudo`.
```

Comment:
What's the change here, exactly?

Reply:
I will update this one. It appears that my tooling prefers to use `-` instead of `+` for bullet points. I will update the tooling for this project.

Reply:
the difference is that I added a blank line at the end of my version. I have removed it as it should not change the page itself.